static/js/containers/Payment.js

```diff
@@ -70,12 +124,38 @@ class Payment extends React.Component {
       <a href="#">Print your statement</a>
     </div>;
   }
+
+  render() {
+    const { selectedKlass } = this.props;
+
+    let renderedKlassDropdown, renderedSelectedKlass;
+    if (this.shouldShowKlassDropdown()) {
+      renderedKlassDropdown = this.renderKlassDropdown();
+    }
+    if (selectedKlass) {
+      renderedSelectedKlass = this.renderSelectedKlass();
+    }
+
+    return <div className="payment">
+      <h3 className="intro">{SETTINGS.user.full_name}, Welcome to MIT Bootcamps</h3>
```

Comment:
it seems like `SETTINGS.user.full_name` can be empty:

![empty_thing](https://cloud.githubusercontent.com/assets/6207644/25404369/a3564568-29cd-11e7-87ce-572faf63a514.png)

not sure how you want to handle that, one suggestion:

```js
formatUserGreeting = () => (
  `${SETTINGS.user.full_name}, Welcome to MIT Bootcamps`.replace(/^,\s/, "")
);
```